1.1-schnorr-signatures.ipynb

```diff
@@ -58,9 +59,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Constraint on the private nonce k\n",
+    "### Constraint on the public key, P, and public nonce point, R\n",
     "\n",
-    "To verify a bip-schnorr signature, the verifier needs the computed `s` scalar and the nonce point `R`. To save 32 bytes in the signature, only the x-value of `R` is provided by the signer, from which the verifier can compute the y-value.\n",
+    "BIP-0340 defines a new way of encoding elliptic curve points. To make the encoding of these points as compact as possible, only the x-value of the points are encoded. Points can therefore be encoded in 32 bytes.\n",
```

Comment:
I feel like the description could also be improved a bit more, e.g. that the 32 bytes saved = the y-coordinate, and why this isn't a malleability issue.

nit: s/x-value/x-values or s/are encoded/is encoded
nit: there's 4 "encode"s in this line 😅

Reply:
Cool yeah will update 😅 👍